src/neo-vm/Types/Boolean.cs

```diff
@@ -7,8 +7,8 @@ namespace Neo.VM.Types
     [DebuggerDisplay("Type={GetType().Name}, Value={value}")]
     public class Boolean : StackItem
     {
-        private static readonly ReadOnlyMemory<byte> TRUE = new byte[] { 1 };
-        private static readonly ReadOnlyMemory<byte> FALSE = new byte[] { 0 };
+        private static readonly byte[] TRUE = new byte[] { 1 };
```

Comment:
ReadOnlySpan<byte>?

Reply:
`ReadOnlySpan` is a `ref struct` type, and can be stored in stack only. It can't be a field of a class.